src/MoneyParser.php

```diff
@@ -12,8 +12,8 @@
     /**
      * Parses a string into a Money object (including currency).
      *
-     * @param string      $money
-     * @param string|null $forceCurrency
+     * @param string     $money
+     * @param mixed|null $forceCurrency
```

Comment:
I would like to see Currency|string|null instead here. Better to be explicit about return types.

Reply:
Good catch. I've updated this.